elasticsearch/run.sh

```diff
@@ -216,4 +216,8 @@ verify_or_add_index_templates() {
 
 verify_or_add_index_templates &
 
+if [ -v HEAP_DUMP_LOCATION ]; then
+    info Setting heap dump location "$HEAP_DUMP_LOCATION"
+    export JAVA_OPTS="${JAVA_OPTS:-} -XX:HeapDumpPath=$HEAP_DUMP_LOCATION"
```

Comment:
ok.  then I suggest something like this - first, remove the `if/fi` lines, then
```
HEAP_DUMP_LOCATION=${HEAP_DUMP_LOCATION:-/elasticsearch/persistent/hdump.prof}
info Setting heap dump location "$HEAP_DUMP_LOCATION"
export JAVA_OPTS="${JAVA_OPTS:-} -XX:HeapDumpPath=$HEAP_DUMP_LOCATION" 
```
Because we need this to always be set